README.md

```diff
@@ -144,3 +144,42 @@ The following is the request flow (seen in blue in the architecture diagram abov
     ```
 
 1. You can see the pods with `kubectl get pods.`
+
+## Uninstalling the component
+1. Remove the demo application
+    ```
+    kubectl delete -f test/app/service.yml
+    ```
+2. If you previously set every service to respond to the `Prefer: respond-async` header, remember to remove this setting by updating the ingress.class to null or by updating the ingress.class to the ingress.class you would like to use, for example `kourier`.
+    ```
+    kubectl patch configmap/config-network \
+    -n knative-serving --type merge \
+    -p '{"data":{"ingress.class":null}}'
+    ```
+
+    ```
+    kubectl patch configmap/config-network \
+    -n knative-serving \
+    --type merge \
+    -p '{"data":{"ingress.class":"kourier.ingress.networking.knative.dev"}}'
+    ```
+3. Remove the producer component
+    ```
+    ko delete -f config/async/100-async-producer.yaml
+    ```
+4. Remove the `RedisStreamSource` and tls secret
+   ```
+   kubectl delete -f config/async/100-async-redis-source.yaml
+   kubectl delete -f config/async/tls-secret.yaml
+   ```
+5. Remove the redis setup - make sure you are using this command in the `eventing-redis` project
+   ```
+   ko delete -f source/config
+   ```
+6. Remove the consumer and async controller components - back in the async component project.
+   ```
+   ko delete -f config/ingress/controller.yaml
+   ko delete -f config/async/100-async-consumer.yaml
+   ```
+
```

Comment:
I would also add a small section `kubectl get pods` or similar to verify these pods are deleted. So maybe `kubectl get pods | grep ...` or something

Reply:
Added instructions to check at each step.  Let me know if you think that's too many. Perhaps it belongs in its own section at the end with all of the commands grouped together.